src/components/letters/Letters.jsx

```diff
@@ -31,40 +34,32 @@ const Letters = () => {
     'Z',
   ];
 
-  
   const [guesses, setGuesses] = useState([]);
- 
-    const checkLetter = (e) => {
-      const letter = e.target.value;
-      console.log(letter);
-      return setGuesses(guesses.concat(letter)); 
-    };
 
-    
+  const checkLetter = (e) => {
+    let letter = e.target.value;
+    socket.emit('guessLetter', letter);
+    console.log(letter);
+    return setGuesses(guesses.concat(letter));
```

Comment:
Eftersom letter är en string(?) så borde kanske push användas istället för concat? 

```suggestion
    return setGuesses(guesses.push(letter));
```

Reply:
> 
> 
> Ser riktigt bra ut! Spännande att få in all den här funktionaliteten!
> 
> En sak jag funderar på är om vi kan förtydliga namnen på `WordDiv `och `Letter`? Tänkte så att det blir tydligt att `Letter `också innehåller funktionaliteten att gissa en bokstav och ` WordDiv` funktionaliteten att hantera en gissning (rätt/fel bokstav). Alternativt att vi skapar en ny komponent som heter exempelvis `MakeAMove` där vi lägger över all funktionalitet.

Det låter som en jättebra idé! Ska se om jag kan fixa det under dagen :)

Reply:
> 
> 
> Eftersom letter är en string(?) så borde kanske push användas istället för concat?

Det borde det tycker jag också. Commitar det :)